### DISCUSSION TITLE
repeat PFTs in config.xml and .css file—how are they matched?

### DISCUSSION THREAD

**Discussion by Aariq** [EDmodel/ED2#377] [Q&A] [answered]
I'd like to do a run with `IED_INIT_MODE=6` to start with existing vegetation.  I also want to use some ED2 PFT numbers more than once to represent different species by using config.xml.  But it seems like there's no way to connect those two.  What happens if a .css file has two cohorts for, say, PFT 10 and the config.xml has two entries for PFT 10?  Are they matched up in some way?

**Comment by xiangtaoxu** [comment, marked as answer]
I don't think so. If you want to run multiple species at the same time, they need to be assigned as different PFTs in ED2, at least in this current version.
